.github/workflows/build-ci-develop.yml

```diff
@@ -0,0 +1,53 @@
+name: Build-CI-Develop
+on:
+  pull_request:
+    branches: [develop]
+  push:
+    branches: [develop]
+jobs:
+  build:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Rust Dependecies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      #       - name: Install toolchain
+      #         run: ./install_rust.sh
+      - name: Build Project
+        if: github.event_name == 'push'
+        run: cargo build --release
+      - run: cargo clean
+      - name: Test Project
+        run: cargo test
+      - name: Run Cargo Trampulin
+        if: github.event_name == 'push'
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.15.0'
+          args: '--exclude polkadex-node node-polkadex-runtime polkadex-primitives  --workspace'
```

Comment:
why are we excluding these?

Reply:
this was the original code. No idea actually. I thought you added. :)

Reply:
removed exlcudes